e2e/tests/datasets/dataset-create.spec.ts

```diff
@@ -75,10 +75,13 @@ describe('Dataset test', () => {
     await datasetCard.selectSnowmanMenu(Option.Edit);
     await waitWhileLoading(page);
 
-    await datasetPage.selectCohorts(['All Participants']);
-    await datasetPage.clickAnalyzeButton();
-
+    const datasetEditPage = new DatasetEditPage(page);
```

Comment:
Created new `dataset-edit-page.ts` class because clicking `Edit` option opens the `Edit Dataset` page.